src/libcore/ops/arith.rs

```diff
@@ -230,17 +200,19 @@ sub_impl! { usize u8 u16 u32 u64 u128 isize i8 i16 i32 i64 i128 f32 f64 }
 
 /// The multiplication operator `*`.
 ///
+/// Note that `RHS = Self` by default, but this is not mandatory.
```

Comment:
I might say "`RHS` is `Self`" instead, because the `=` is real syntax and I find this wording slightly confusing because of it.

i might also be a weirdo